modules/client-handler/src/main/java/org/apache/ignite/client/handler/JdbcQueryEventHandlerImpl.java

```diff
@@ -498,7 +498,7 @@ <T> CompletableFuture<T> doInSession(SessionAwareAction<T> action) {
                             return action.perform(newSessionId);
                         }
 
-                        return CompletableFuture.failedFuture(IgniteExceptionMapperUtil.mapToPublicException(error));
+                        return CompletableFuture.failedFuture(IgniteExceptionMapperUtil.mapToPublicSqlException(error));
```

Comment:
as far as I see, none of the tests are affected by this change. So, why have you changed this?

Reply:
agree